docs/flink-connector/flink-authentication-with-gravitino.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Flink authentication with Gravitino server"
+slug: /flink-connector/flink-authentication
+keyword: flink connector authentication oauth2 kerberos
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Flink connector supports `simple`, `oauth2`, and `kerberos` authentication when accessing the Gravitino server.
+
+| Property                   | Type   | Default Value | Description                                                                                                                              | Required | Since Version |
+|----------------------------|--------|---------------|------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| gravitino.client.auth.type | string | (none)        | When explicitly set, only `oauth` is supported. If unset, Flink selects Kerberos or simple authentication based on its security settings. | No       | 1.2.0         |
+
+## Simple mode
+
+In simple mode, the username originates from Flink. The resolution order is:
+1. `HADOOP_USER_NAME` environment variable
+2. The logged-in OS user
+
+## OAuth2 mode
+
+In OAuth2 mode, configure the following settings to fetch an OAuth2 token to access the Gravitino server:
```

Comment:
How should user configurate belew settings? `gravitino.client.oauth2.serverUri` or `table.catalog-store.gravitino.gravitino.client.oauth2.serverUri`? could you provide an example?